src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -89,6 +89,18 @@ public final class Constants {
     // Used as a value for <sourceDirectory> for the Java template pom.xml
     public static final String STUDENT_WORKING_DIRECTORY = ASSIGNMENT_DIRECTORY + "src";
 
+    public static final String USER_FIRST_NAME_AFTER_SOFT_DELETE = "Deleted";
+
+    public static final String USER_FIRST_NAME_AFTER_SOFT_DELETE_DE = "Gelöschter";
+
+    public static final String USER_LAST_NAME_AFTER_SOFT_DELETE = "User";
+
+    public static final String USER_LAST_NAME_AFTER_SOFT_DELETE_DE = "Benutzer";
+
+    public static final String USER_EMAIL_DOMAIN_AFTER_SOFT_DELETE = "@user.deleted";
+
+    public static final String USER_EMAIL_DOMAIN_AFTER_SOFT_DELETE_DE = "@benutzer.gelöscht";
```

Comment:
I don't like these translations. You set them based on the admin language, which doesn't make much sense because the translated values will not adapt to a user's client language anyways. Worst case we have a german "gelöschter Benutzer" for English speaking people in the client.

So either we set a translation key here and translate it everywhere in the client like Johannes suggested above, or, as I understand why you don't want to do that, just stick to English as that's the default language most people will understand. In any case I would not base it on the admin language.